config/helm/delete-leader-election.yaml

```diff
@@ -0,0 +1,44 @@
+# We are not using leader election in our current v0.3.x release, since we are
```

Comment:
Patch to delete all manifests related to leader election, since we are not using it ATM.